Sources/SpeziFoundation/Misc/TimeoutError.swift

```diff
@@ -18,11 +18,24 @@ public struct TimeoutError {
 
 extension TimeoutError: LocalizedError {
     public var errorDescription: String? {
-        String(localized: LocalizedStringResource("Timeout", bundle: .atURL(Bundle.module.bundleURL)))
+    #if os(Linux)
+        return "Timeout"
+    #else
+        String(
+            localized: LocalizedStringResource(
+                "Timeout",
+                bundle: .atURL(Bundle.module.bundleURL)
+            )
+        )
+    #endif
     }
 
     public var failureReason: String? {
+    #if os(Linux)
+        return "The operation timed out."
+    #else
         String(localized: LocalizedStringResource("The operation timed out.", bundle: .atURL(Bundle.module.bundleURL)))
+    #endif
```

Comment:
I think we can import the open-source version of foundation for Linux Builds and then use sting localization. The implementation seems to support it: https://swiftpackageindex.com/swiftlang/swift-foundation; wondering if an explicit import of Foundation would fix this?